common/config/.golangci.yml

```diff
@@ -329,6 +329,15 @@ issues:
     - linters:
         - staticcheck
       text: "SA1019: grpc.WithReturnConnectionError"
+    - linters:
```

Comment:
was this supposed to be changed in common-files repo?

Reply:
should be in `tools/golangci-override.yaml` probably

Reply:
@mikemorris